cfgov/unprocessed/js/molecules/Autocomplete.js

```diff
@@ -86,6 +85,7 @@ function Autocomplete( element, opts ) {
    */
   function init() {
     if ( !atomicHelpers.setInitFlag( _dom ) ) {
+      let UNDEFINED;
       return UNDEFINED;
```

Comment:
For this use, we could just `return`, but I think it's slightly clearer to set what's returned.